README.md

```diff
@@ -1,27 +1,22 @@
-<!-- Describe this theme/component in one or two sentences -->
-
 A clean & modern theme with a handful of theme-components included to enhance your forum!
 
-<!-- Add screenshots (if applicable) -->
-
 **Light Mode**
 
-![image](https://user-images.githubusercontent.com/5862206/214545439-85410d82-9565-4e00-8b23-b2d69a0ee1eb.png)
+![light mode](screenshots/light.png)
 
 **Dark Mode**
 
-![image](https://user-images.githubusercontent.com/5862206/214545506-f44d6165-4f79-416e-a89c-548482d04cea.png)
+![dark mode](screenshots/dark.png)
 
 **Categories Page**
 
-![image](https://user-images.githubusercontent.com/5862206/214545568-76d38925-55a6-4359-b1c6-bf1010706132.png)
+![categories page](images/categories-page.png)
 
 This theme includes a handful of components to enhance your forum as well.
 
 - Dark Light Scheme Toggle
-- Clickable Topics
+- [Clickable Topics](https://meta.discourse.org/t/clickable-topic/183339)
 - Discourse Loading Slider
-- Discourse Search Banner
 - Modern Category + Group Boxes
```

Comment:
only Clickable Topics has Meta topic
Modern Category + Group Boxes - has only github repo (see above `about.json:7` - shall we include it?

not sure we even have `Dark Light Scheme Toggle` and `Discourse Loading Slider` used in this theme, unless these were ported into core

Reply:
Only the components listed as dependencies in about.json need to be listed here:

https://github.com/discourse/discourse-air/blob/67f68e442e856d4a4293c6cea61bfaea6f7ea9be/about.json#L5-L9

You are removing search banner, so only clickable topics and the category boxes need to be linked. I would speak to @jordanvidrine further about this if you need to, he is the creator of the theme.

One thing we should do is list https://github.com/discourse/discourse-minimal-category-boxes as the dependency, not https://github.com/jordanvidrine/discourse-category-group-boxes in about.json